cli/config_file.rs

```diff
@@ -266,10 +266,33 @@ impl Serialize for TsConfig {
   }
 }
 
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct LintRulesConfig {
+  pub tags: Option<Vec<String>>,
```

Comment:
@dsherret sorry I missed this comment. There's no single issue in `deno_lint` repo at the moment, but the basic idea is that there would be tags like "js-only" or "ts-only". I'm fine with skipping tags in the first pass, but that means users would only be able to exclude and include rules one by one. I guess it might be fine for first pass after all.

Reply:
I guess if this whole feature is unstable then adding tags is not a big deal as we can remove it if necessary once stabilizing.